### ISSUE TITLE
"Always open on this container" disappeared

### ISSUE THREAD

**Issue by comzeradd** [mozilla/testpilot-containers#734]
Not sure if this has been reported again, but since some days now the option to open a specific domain on a specific container has disappeared from Nightly. That also resulted on loosing such previous configurations. Domains I have set to open on specific containers don't do anymore.

**Comment by jonathanKingston** [comment]
Closing as a duplicate of: #725 the release should be cut today.
The extension is disabled despite you getting native nightly experience. The config shouldn't be lost when the update happens.

**Comment by markrcote** [comment]
I'm confused--is this supposed to be fixed now that #725 is also closed? I was seeing the menu option intermittently disappear, but then I just disabled and re-enabled the Test Pilot experiment and the option seems to have permanently disappeared. Is this expected?

**Comment by jonathanKingston** [comment]
It should work for this, are you using a container tab to actually do the
assignment from?
If so do you have more details on your environment?
Thanks

**Comment by markrcote** [comment]
Yes, I am. And just now it started appearing again in some tabs, but not others. I currently have two different sites open in two different containers. On one I see the option; on the other, I don't.

I'm running the latest Nightly on Windows 10 (Dell XPS13).

**Comment by markrcote** [comment]
It seems that sites that I had pinned to particular containers some time ago are the ones that don't have the "Always Open in this Container" option--nor do they open in the proper (or any) container anymore.
